docs/sources/reference-pyroscope-architecture/about-grafana-pyroscope-architecture/index.md

```diff
@@ -59,6 +59,9 @@ The [query-scheduler]({{< relref "../components/query-scheduler" >}}) maintains
 
 The [querier]({{< relref "../components/querier" >}}) act as workers, pulling queries from the queue in the query-scheduler. The queriers connect to the ingesters to fetch all the data needed to execute a query. For more information about how the query is executed, refer to [querier]({{< relref "../components/querier.md" >}}).
 
+Depending on what time window is select the querier involves [ingesters]({{< relref "../components/ingester" >}}) for recent data, and [store-gateways]({{< relref "../components/store-gateway" >}}) for data from long-term storage 
```

Comment:
This is a minor change to add a missing comma for the opening phrase, and add a missing period. 
```suggestion
Depending on the time window is selected, the querier involves [ingesters]({{< relref "../components/ingester" >}}) for recent data and [store-gateways]({{< relref "../components/store-gateway" >}}) for data from long-term storage.
```